src/main/java/seedu/commands/BudgetCommand.java

```diff
@@ -36,25 +36,26 @@ public BudgetCommand(String fullCommand) {
      */
     @Override
     public void execute(IncomeList incomes, SpendingList spendings) {
-        String[] userInputWords = fullCommand.split(" ", 3);
+        String[] arguments = fullCommand.split(" ", 3);
         try {
-            if (userInputWords.length != 3) {
+            if (arguments.length != 3) {
```

Comment:
Actually not sure if most of our command that check for argument numbers, they magic numbers.
Would arguments.length != CORRECT_NUMBER_OF_ARGUMENTS be better.